style.css

```diff
@@ -148,10 +148,72 @@ body {
   color: #fff;
   border-radius: 4px;
   background: #0056EC;
+  border: 0;
+  transition: background .2s ease-out 0s;
+  outline: none;
 }
 
 .main__button:hover {
   cursor: pointer;
   background: #1e90ff;
-  transition: 1s;
 }
+
+.footer {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  max-width: 80%;
+}
+
+.footer__circle {
+  position: relative;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  opacity: .7;
+  margin-right: 4px;
+  background: #c8dbfb;
+}
+
+.footer__circle:last-child {
+  margin-right: 0;
+}
+
+@media (max-width: 1366px) {
+  .wrapper {
+    max-width: 68%;
+}
+
+  .header {
+    margin-bottom: 65px;
+  }
+
+  .menu__item {
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .main__title {
+    padding-top: 40px;
+    font-size: 50px;
+    line-height: 66px;
+  }
+
+  .main__description {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 21px;
+  }
+
+  .main__button {
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  .main {
+    padding: 0 50% 50px 0;
+    background-size: contain;
+    margin-bottom: 90px;
+  }
+}
+  
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tni-htmlcss2.0/pullRequest?prid=6206132)